testcommon/testInitializer_vmoutput.go

```diff
@@ -1,13 +1,15 @@
 package testcommon
 
 import (
+	"bytes"
 	"fmt"
 	"math/big"
 	"testing"
 	"unicode"
 
 	"github.com/ElrondNetwork/arwen-wasm-vm/v1_4/arwen"
 	"github.com/ElrondNetwork/elrond-go-core/data/vm"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
```

Comment:
Maybe use camel case instead of snake case for all files?

Reply:
Renamed.